internal/build/job/maker.go

```diff
@@ -205,18 +201,18 @@ func (m *maker) getHashAnnotationValue(ctx context.Context, configMapName, names
 }
 
 func volumes(imageRepoSecret *v1.LocalObjectReference, buildConfig *kmmv1beta1.Build) []v1.Volume {
-	volumes := []v1.Volume{dockerfileVolume(dockerfileVolumeName, buildConfig.DockerfileConfigMap)}
+	volumes := []v1.Volume{dockerfileVolume(DockerfileVolumeName, buildConfig.DockerfileConfigMap)}
```

Comment:
why do we need those changes for this PR? we don't need to change build at all, just duplicate what is needed into sign, and the do the real refactoring in the next PR. IMHO it will make this PR more clear

Reply:
Removed the changes to `internal/build` from this PR.